packages/transport_webrtc/src/transport.rs

```diff
@@ -45,7 +45,7 @@ mod whip;
 pub enum VariantParams<ES> {
     Whip(RoomId, PeerId),
     Whep(RoomId, PeerId),
-    Webrtc(String, ConnectRequest, Arc<ES>),
+    Webrtc(String, Box<ConnectRequest>, Arc<ES>),
```

Comment:
Why we need to wrapping inside Box? Is it over enum size?

Reply:
@giangndm Yes, clippy complained that the Enum member sizes had a wider gap:

```
warning: large size difference between variants
  --> packages/transport_webrtc/src/transport.rs:45:1
   |
45 | / pub enum VariantParams<ES> {
46 | |     Whip(RoomId, PeerId),
   | |     -------------------- the second-largest variant contains at least 48 bytes
47 | |     Whep(RoomId, PeerId),
48 | |     Webrtc(String, ConnectRequest, Arc<ES>),
   | |     --------------------------------------- the largest variant contains at least 264 bytes
49 | | }
   | |_^ the entire enum is at least 264 bytes
   |
   = help: for further information visit https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant
   = note: `#[warn(clippy::large_enum_variant)]` on by default
help: consider boxing the large fields to reduce the total size of the enum
```

Reply:
@giangndm Should I change it back and silence the clippy instead?

Reply:
@dhilipsiva Yes, I think we shouldn't to use Box here for don't confusing other developer. With box some guy will ask why it is needed.

Reply:
Removed the Box now. And silenced clippy.